### DISCUSSION TITLE
本地运行没有问题，部署到Github时出现Cannot use import statement outside a module异常

### DISCUSSION THREAD

**Discussion by le0zh0u** [vuepress/core#1087] [Q&A] [answered]
之前一段时间都是正常的，但是最近几天将vuepress站点部署到github pages时，执行Action出现了Cannot use import statement outside a module异常

具体异常如下：
<img width="1307" alt="image" src="https://user-images.githubusercontent.com/4425506/190372972-010dff57-c02e-4344-8e42-2433a1e5e277.png">

最近一次执行：
https://github.com/le0zh0u/highlights/actions/runs/3059411402/jobs/4936757119

尝试了：
删除package-lock.json，以防是依赖问题
重新build后上传，复现
确认最近一次成功babel的版本和现在的版本一致

本地清理了yarn缓存后，重新执行build，出现相同异常，然而不知道如何解决。想问下是否有类似情况，以及该如何处理

20220917更新：
我在本地yarn cache clean之后尝试重新install，package中的依赖是`"vuepress": "^2.0.0-beta.51"`，但是在安装时显示的都是`2.0.0-beta.50-pre.1: deprecated`，请问下这个是版本选择问题吗？

**Comment by Mister-Hope** [comment, marked as answer]
Try `"vuepress": "2.0.0-beta.51"` 

CC @meteorlxy when will you delete that deprecate version ?

**Reply by le0zh0u** [reply]
Thanks, it fixed
